bitrise.yml

```diff
@@ -223,6 +223,7 @@ workflows:
         - pull_request_repository_url: ""
         - branch_dest: ""
         - clone_depth: ""
+        - fetch_tags: "no" # this case the step falls back to related tag fetching
```

Comment:
Does it fall back?

Reply:
removed, this comment reflected the previous state of the PR